xml_inscriptions/transcribed/NY.Ith.CU.L.CCC.741-mdn.xml

```diff
@@ -0,0 +1,188 @@
+                            target="https://digital.library.cornell.edu/catalog/ss:172695">Cornell
+                            Cast Collection</ref>
+                    </bibl>
+                    <bibl>
+                        <ref type="external"
+                            target="http://www.edr-edr.it/edr_programmi/res_complex_comune.php?%20do=book&amp;id_nr=edr090272">EDR090272</ref>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for US Epigraphy controlled values</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc/>
+        <revisionDesc>
+            <change when="2025-02-15" who="Norah Mezey-McMahon">Initial encoding</change>
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.Ith.CU.L.CCC.741-mdn.jpg"/>
+        </surface>
+    </facsimile>
+    <text>
+        <body>
+            <div type="edition">
+                <ab>
+                    <lb n="1"/>
+                        <persName>
+                            <name type="praenomen" key="Marcus">
+                                <expan><abbr><unclear>M</unclear></abbr><ex>arcus</ex></expan>    
+                            </name>
+                            <name type="gentilicium" key="Valerius">
+                                <supplied reason="lost">Valeriu</supplied>s
+                            </name>
+                            <persName>
+                                <name type="praenomen" key="Marcus">
+                                    <expan><abbr>M</abbr><ex>arci</ex></expan>
+                                </name>
+                                <name type="praenomen" key="Lucius">
+                                    <expan><abbr>L</abbr><ex>ucius</ex></expan>
+                                </name>
+                                <name type="praenomen" key="Publius">
+                                    <expan><abbr>P</abbr><ex>ubli</ex></expan>
+                                </name> 
+                            </persName>
+                            <expan><abbr><am><g type="Gaiae"/></am></abbr><g type="mulieris"/></expan>
+                            <expan><abbr>l</abbr><ex>ibertus</ex></expan>
+                     <lb n="2"/>
+                            <name type="cognomen" key="Anteros">
+                                Anteros
+                            </name>
+                            <name type="cognomen" key="Asiaticus">Asiaticus</name>
+                        </persName>,
+                        <choice><reg>Se</reg><orig>Vi</orig></choice>vir,
```

Comment:
VIvir is one of the standard ways of expressing this epigraphically; I'm not sure I would regularize it here, and I'd probably leave it is.